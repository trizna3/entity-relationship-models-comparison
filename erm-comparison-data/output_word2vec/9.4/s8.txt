line-lines
vehicle type-vehicle type
stop-stops
vehicle-vehicles
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-vehicle type-lines
