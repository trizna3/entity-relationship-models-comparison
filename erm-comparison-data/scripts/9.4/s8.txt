entity set stops {
	"name"
};

entity set lines {
	"vehicle type"
};

entity set vehicles {
	"type"
	"id"
};

association "duration till stop" between {
	stops MANY
	lines MANY
}{
	"time"
};


association "rides on" between {
	vehicles MANY
	lines MANY
}{
	"takeoff time"
};