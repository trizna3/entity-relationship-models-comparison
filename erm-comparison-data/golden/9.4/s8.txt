vehicle-vehicles
vehicle type-vehicle type
line-lines
stop-stops
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-vehicle type-lines
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-type-vehicles